paket.lock

```diff
@@ -32,6 +39,7 @@ NUGET
       System.Threading.Thread (>= 4.3)
       System.Threading.ThreadPool (>= 4.3)
     FSharp.Core (5.0.1)
+    FSharp.Core (5.0)
```

Comment:
Duplicate?

Reply:
that's super weird, let me recheck this

Reply:
yep, I just flubbed the paket.dependencies in a way that included both. this should be fixed now.